### ISSUE TITLE
[Bug] can not download openvino toolkit from intel download page

### ISSUE THREAD

**Issue by softsweetengineer** [openvinotoolkit/openvino#7667]
when i try to download openvino distribution from https://software.seek.intel.com/openvino-toolkit , It just jumps to an error page that I can not download the latest version at all.

Can you give me another download link for windows to install this toolkit?

**Comment by brmarkus** [comment]
Can you try via this entry point, please?
https://software.intel.com/en-us/openvino-toolkit

**Comment by softsweetengineer** [comment]
Duplicate of # 
Tried and failed, 
1. click "choose & download." 
2. it jumps to ` https://software.seek.intel.com/openvino-toolkit`
3. filled the tables, and click "download"
4. jump to https://registrationcenter.intel.com/en/home/error/999999/

Download not started.

**Comment by softsweetengineer** [comment]
use F12 can see some API errors happened when click download:


`(index):13 Access to XMLHttpRequest at 'https://signin.intel.com/?TYPE=33554433&REALMOID=06-1a08b802-d9f5-4485-975b…ET=$SM$https%3a%2f%2fsfederation%2eintel%2ecom%2ffederation%2fstatus%2easp' (redirected from 'https://sfederation.intel.com/federation/status.asp') from origin 'https://registrationcenter.intel.com' has been blocked by CORS policy: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'. The credentials mode of requests initiated by the XMLHttpRequest is controlled by the withCredentials attribute.
(index):13 HEAD https://signin.intel.com/?TYPE=33554433&REALMOID=06-1a08b802-d9f5-4485-975b…ET=$SM$https%3a%2f%2fsfederation%2eintel%2ecom%2ffederation%2fstatus%2easp net::ERR_FAILED 302
assetLoader.min.js:114 GET https://www.intel.com/content/www/us/en/404.html net::ERR_ABORTED 404
secondaryNav.min.js:165 The IGHF breadcrumb is ready to use
assetLoader.min.js:114 GET https://www.intel.com/content/www/us/en/404.html net::ERR_ABORTED 404`

**Comment by yury-gorbachev** [comment]
You might want to check this page: https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_images.html
binaries should be pretty much the same...

**Comment by softsweetengineer** [comment]
> You might want to check this page: https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_images.html
> binaries should be pretty much the same...

After trying to install binary version openvino, I faced some problems when converting yolov4 into IR following document,
my envernment is:
sys : WINDOWS 10-64
python: Python 3.6.13 :: Anaconda, Inc.

I tried to convert yolov4 to IR following the document, and when I tried to run mo.py, I got some problems:
ERROR:root:Could not find the Inference Engine or nGraph Python API.
Consider building the Inference Engine and nGraph Python APIs from sources or try to install OpenVINO (TM) Toolkit using "install_prerequisites.sh"

here is what I have tried: 
I tried using both conda and pip to install the openvino with two command:
1.  pip install openvino
2.  conda install openvino-ie4py -c intel
after running those commands, I tried to run install_prerequisites.bat
and get error like:
[ WARNING ] Failed to import Inference Engine Python API in: PYTHONPATH
[ WARNING ] No module named 'ngraph.frontend'
these is the full output: 
[error.txt](https://github.com/openvinotoolkit/openvino/files/7236142/error.txt)

**Comment by softsweetengineer** [comment]
I have received the directly link to download the openvino installer from intel, problem solved.

THANK YOU FOR YOUR HELP!
the link is blown: 
https://registrationcenter-download.intel.com/akdlm/irc_nas/18097/w_openvino_toolkit_p_2021.4.689.exe

**Comment by jgespino** [comment]
We identified and fixed a bug in our registration system last week that may have caused a disruption to your experience downloading the Intel® Distribution of OpenVINO toolkit. This issue has now been remedied and we invite you to return to complete your download. For an optimal download experience, we recommend removing any ad blockers.
